docs/docs/studio.md

```diff
@@ -6,6 +6,15 @@ description: RedwoodJS Studio is a package used during development to gain runti
 
 RedwoodJS Studio is a package used during development to gain runtime insights into a project.
 
+We want Studio to be an essential tool for every RedwoodJS developer.
```

Comment:
I think we should get rid of this section on what we want Studio to be, especially since we don't have a timeline or in the event that we change our minds.